modules/standard/OS.chpl

```diff
@@ -34,6 +34,7 @@
 
  */
 module OS {
+  private use CTypes;
```

Comment:
Not that it's necessarily problematic, but is this `use` required?  I think OS is still effectively an empty module and `POSIX` has its own `use CTypes`